### ISSUE TITLE
Environment variables unavailable in inlineScript

### ISSUE THREAD

**Issue by oliwheeler** [Azure/cli#7]
Currently, only the GITHUB_WORKSPACE environment variable is set.
The other [default environment variables](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables) are not available.

Environment variables set by the user in the workflow YAML are also not available.

The documentation regarding [create a Docker container action](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-docker-container-action) shows that the environment variables are passed to the container [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-docker-container-action#testing-out-your-action-in-a-workflow)

__Expected behaviour:__
  * All default environment variables are available in inlineScript
  * All user set environment variables are available in inlineScript

**Comment by julie-ng** [comment]
@jalajmsft - why is this closed? Expected behavior as posted by original user is still **NOT** resolved:

> All user set environment variables are available in inlineScript

Yep, I need this behavior too because I want to reuse the commands and pass variables depending on environment (dev vs production). This is what I am using and it does not work. 

```yaml
    - name: "Azure CLI: upload assets to blob storage"
      uses: azure/CLI@v1
      env:
        STORAGE_ACCOUNT: "mystorageaccount"
        STORAGE_CONTAINER: "assets"
      with:
        inlineScript: |
          echo "doesn't work $STORAGE_ACCOUNT"
          echo "doesn't work either ${{ env.STORAGE_ACCOUNT }}"
          echo "so of course below fails"
          az storage blob upload-batch \
            --account-name ${{ env.STORAGE_ACCOUNT }} \
            --source ./public/assets \
            --destination ${{ env.STORAGE_CONTAINER }}
```

**Comment by julie-ng** [comment]
Um, no idea why @ing @jalajmsft unassigned them from issue… sorry 🤷‍♀️

**Comment by julie-ng** [comment]
Nevermind, this was fixed in v1.0.3 release, but the commits never referenced this issue
https://github.com/Azure/cli/releases/tag/v1.0.3

**Comment by branostupak** [comment]
What's the proper way to use ENV variables in `inlineScript`?
